frontend/workflows/projectSelector/src/card.tsx

```diff
@@ -11,7 +11,7 @@ interface CardProps {
 }
 
 const Card = ({ avatar, title, error, children }: CardProps) => (
-  <Grid item xs={12} sm={7}>
+  <Grid item xs={12} sm={6}>
```

Comment:
The max size of the grid is 12 right? Giving each card 6 width to fill up? And below does `flex={1}` for both box's just fill that 6 width up?

Reply:
yeah, we want them to display side by side. The flex below for the second box fills up the container.